Dockerfile

```
@@ -44,6 +44,7 @@ RUN \
   # clean jhipster folder
   rm -Rf /home/jhipster/generator-jhipster/node_modules && \
   # install jhipster
+  cd /home/jhipster/generator-jhipster && npm install && \
```

There are two npm install executions, see next line

---

The new line is to install missing dependencies, the 2nd line (existing one) is to install the generator.
I don't understand what happened since 2 days but without that, the Docker image is broken. Maybe new version of npm?
If you have a better idea, you're welcome :)

---

Why don't remove `rm -Rf /home/jhipster/generator-jhipster/node_modules && \`
and change to:
```suggestion
  cd /home/jhipster/generator-jhipster && npm ci && \
```

I think this is related to npm 7 been declared stable, npm install -g seem to behave just like npm link.

---

Thanks @mshima : I apply your suggestion